### ISSUE TITLE
cannot update the description of a task

### ISSUE THREAD

**Issue by DinoMarlir** [hudy9x/namviek#268]
**Describe the bug**
You can't update the description of a task. It gets reset every time I create a subtask or leave the view

**To Reproduce**
you can see it in the video

**Expected behavior**
That the description gets updated

**Screenshots**
[Video (how to reproduce this issue)](https://github.com/user-attachments/assets/99184ae2-d1e4-4c36-bf29-061aa04ac326)
**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: safari
 - Version: 18.0.1 (20619.1.26.31.7)

**Comment by DasSharkk** [comment]
Works fine for me on the same instance though, just tested it in Safari and in Arc, both work perfectly fine for me @DinoMarlir

**Comment by DinoMarlir** [comment]
No, I don't see the success message. I get some errors in the console, seems to be an issue with the date serialization.

<img width="1084" alt="image" src="https://github.com/user-attachments/assets/275b49cd-3ade-4e0f-b8e2-72583ff7c6ef">

<img width="1113" alt="image" src="https://github.com/user-attachments/assets/b19e0503-8ab1-4b70-9227-de2c62ad6580">

**Comment by hudy9x** [comment]
Tried several time and everything work well
But found that planned Date not to be used actually
So I'll remove plannedDate
